src/attached_probe.cpp

```diff
@@ -381,54 +381,38 @@ static uint64_t resolve_offset(const std::string &path,
   return bcc_sym.offset;
 }
 
-static void check_alignment(std::string &path,
-                            std::string &symbol,
-                            uint64_t sym_offset,
-                            uint64_t func_offset,
-                            bool safe_mode,
```

Comment:
I think we'd still want to keep the `safe_mode` option for this function. Normally we don't want to allow unaligned probes (although we shouldn't `abort()`), but there might be cases where we can't correctly determine instruction boundaries and users should be able to override our checks.

Reply:
>  Normally we don't want to allow unaligned probes

This PR doesn't change that. If the alignment isn't valid we don't attach the probe. This is just utilizing the MissingProbes config to allow the user to specify how they want to handle this failure. The default now, instead of aborting, is to `LOG(WARN)` and not attach the probe. This is so that when attaching to multiple probes with a glob the whole script doesn't fail.